src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
             }
           }
         },
+        // TODO: Should this be privateMedicalRecordsRelease or privateRecordsRelease? 
+        privateMedicalRecordsRelease: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-release',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          depends: (formData, index) => {
+            const hasRecords = _.get(`disabilities.${index}.view:privateMedicalRecords`, formData);
+            // TODO: enable once previous chapter merged
+            // const requestsRecords = !_.get(`disabilities.${index}.view:uploadPrivateRecords`, formData);
+            const requestsRecords = true;
+            return hasRecords && requestsRecords;
+          },
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:description': 'Please let us know where and when you received treatment. We’ll request your private medical records for you. If you have your private medical records available, you can upload them later in the application',
+                treatments: {
+                  'ui:options': {
+                    itemName: 'Record',
+                    viewField: () => {
+                      return (<div>hello</div>);
```

Comment:
:red_circle: :robot: HELLO, DAVE! :robot: :red_circle: 

...This is part of that clean up you were talking about, yeah?

Reply:
Yep, updated using the treatmentView component.